### PR TITLE
Add policy on expenses for working at home

### PR DESCRIPTION
**Pull request by chrisblackburn**
Add policy on equipment to support home working

(this whole expense section could use some work, but that yak is not for today)

### CODE REVIEW

**Comment by StevenLeighton21** [comment]
Might be nice to clarify how to go about claiming it, ie 'Order what you need and expense through Xero' or 'Provide a delivery address and links to the equipment you need to the people team who will order it'. 

Looks good though

**Review by StevenLeighton21 — APPROVED** [review]

**Comment by leaky** [comment]
I feel like this should somehow tie into the "Work Ready" benefit, or serves as an extension of.

https://github.com/madetech/handbook/blob/master/benefits/work_ready.md

**Comment by chrisblackburn** [comment]
I'll let that get added as a subsequent improvement (the main expenses page from which this is linked does an OKish job of explaining how to make an expense claim)

File: guides/compensation/expenses.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ We provide some examples below of what is okay to expense. We trust you to be ab
...
 - Training costs
-- Computer equipment (new monitor, keyboard, HDMI adapters, etc.)
+- Small computer equipment (HDMI adapters etc.)
 - Amenities for the office (coffee, fruit, biscuits, etc.)
 - Lunch and drinks for a customer
 - Taxi to and from the office for a meeting
 - Taxi home after working late
 - Stationary (post-it notes, notepads, etc.)
-- Conference tickets (speak with a Director or Line Manager if over £150 per ticket)
+- Conference tickets (speak with a Director or Line Manager if over £150 per ticket)]
```
Comment (adrianclay) [inline guides/compensation/expenses.md:17]: Looks like a rogue `]`

**Pull request merged**